src/domains/Invite/api/api.test.ts

```diff
@@ -0,0 +1,167 @@
+/* eslint-disable @typescript-eslint/unbound-method */
+import { AxiosRequestConfig } from 'axios';
+import axios from 'utils/axios';
+import errors from 'utils/errors';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import faker from 'faker';
+import API from './index';
+import { InviteForm, InvitePreview } from '../types';
+
+describe('createInvite', () => {
+    const testEventDateTime = faker.date.future().toUTCString();
+    const testDeliveryTime = new Date();
+    const testFormData: InviteForm = {
+        MoC: faker.name.firstName(),
+        topic: 'Topic',
+        eventDateTime: testEventDateTime,
+        constituentScope: 'state',
+        region: 'test',
+        deliveryTime: testDeliveryTime,
+        townHallId: 'testID',
+    };
+
+    const testPreview: InvitePreview = {
+        sendPreview: true,
+        previewEmail: faker.internet.email(),
+    };
+
+    const testFileData = [
+        `email,fName,lName\n
+    ${faker.internet.email()},${faker.name.firstName()},${faker.name.lastName()}`,
+    ];
```

Comment:
We should probably add more info to this so we don't run into some special case since there's only 1 row's worth of data -- excluding the headers.